release-notes/6.0/linux-packages.md

```diff
@@ -0,0 +1,81 @@
+# Linux package dependencies
+
+.NET 6 has several dependencies that must be satisfied to run .NET apps. The commands to install these libraries are listed for multiple distributions.
+
+Feel free to contribute packages for distributions not (yet) listed in this document, including ones not supported by the .NET Team.
+
+Tips:
+
+- [runtime-deps container images](https://github.com/dotnet/dotnet-docker/tree/main/src/runtime-deps) installs these same packages. You can look at those dockerfiles.
+- [pkgs.org](https://pkgs.org/) is a useful site for searching for packages, to find the one for your distribution.
+
+## Packages
+
+.NET depends on the following packages.
+
+- [GNU C Library (glibc)](https://www.gnu.org/software/libc/libc.html)
+- [GNU C++ Library](https://gcc.gnu.org/onlinedocs/libstdc++/)
+- [GCC low-level runtime library](https://gcc.gnu.org/onlinedocs/gccint/Libgcc.html)
+- [ICU](http://site.icu-project.org/)
+- [Kerberos](http://web.mit.edu/kerberos/)
+- [Open SSL](https://www.openssl.org/)
+- [zlib compression library](https://www.zlib.net/)
+
+If your app relies on `https` endpoints, you'll also need to install `ca-certificates`.
+
+## Alpine 3.13
```

Comment:
We also maintain these lists of dependencies in topics like https://docs.microsoft.com/en-us/dotnet/core/install/linux-alpine of course .. I assume we should keep them in sync. Comparing them for Alpine, it looks this list is missing `bash` which is necessary for `dotnet` to work. Conversely the other list should probably note that `libicu` is optional.

Reply:
Where does dotnet depend on bash?

Reply:
Hmm, when I was trying to run the perf scripts on Alpine recently, it seemed that bash was required to run the CLI. Perhaps I was mistaken and it was something else in the infra?

Reply:
`bash` is not required. For example, the macOS shell is `zsh`. We use `ash` with Alpine and never noticed a problem.

The list on docs is helpful, but hard to use. See: https://docs.microsoft.com/dotnet/core/install/linux-debian#dependencies. I'm going for copy-and-past friendly.

Reply:
It is likely the script was dependent on bash. It is trivially easy to make a script dependent on a shell.

Reply:
I bet you're right (the script probably has bash at the top!). Well, I need to fix the prereq doc for Alpine to remove that.

Reply:
https://github.com/dotnet/docs/pull/23555

https://github.com/dotnet/install-scripts/issues/173